src/main/java/io/github/oliviercailloux/y2018/apartments/apartment/ApartmentFactory.java

```diff
@@ -198,35 +210,24 @@ private static String getRandomAddress(int retry) throws IOException {
 	 */
 	private static String getRandomAddress() {
 		try {
-			return getRandomAddress(0);
-		} catch (IOException e) {
-			LOGGER.error("Problem while getting random address", e);
+			return ApartmentFactory.getOnlineRandomAddress();
+		} catch (ClientErrorException | InvalidObjectException e) {
```

Comment:
Il faut annoncer ces exceptions sur `getOnlineRandomAddress`, si elles sont considérées comme « attrapables ».

Reply:
Fix in a80a8f27fd1a55806ab72635368de4f7dac3f16d
Nous annonçons `ClientErrorException` bien qu'il s'agit d'une `RuntimeException` levé par JAX-RS